src/ui/components/Library/Team/View/Tests/Overview/Execution.tsx

```diff
@@ -27,7 +27,11 @@ export function Execution({ execution }: { execution: TestExecution }) {
   }
 
   return (
-    <div data-test-id="ExecutionItem" className="flex flex-col px-2">
+    <a
+      data-test-id="ExecutionItem"
+      className="flex flex-col px-2"
+      href={shouldCollapse ? `/recording/${sortedRecordings[0].id}` : undefined}
```

Comment:
Instead of conditionally setting the `href`, let's swap component type between `div` and `a`